front-end/src/components/HeatChart.js

```diff
@@ -9,22 +9,23 @@ import './HeatChart.css';
 
 class HeatChart extends Component {
   componentDidMount() {
-    this.props.fetchHeatChartData();
+    const { startDate, endDate } = this.props.datePickerData;
+    if (startDate !== null && endDate !== null) {
+      this.props.fetchHeatChartData(startDate, endDate);
```

Comment:
viết thế này thì cái `this.props.fetchHeatChartData(startDate, endDate);` không bao giờ được chạy luôn? 
a muốn là vẫn phải hiển thị chart bình thường cơ mà

Reply:
cái đoạn này e đang muốn nó sẽ chạy khi mình thay đổi date mặc định =))

Reply:
theo a thì bỏ luôn componentDidMount hả =))

Reply:
để e sửa lại đoạn này